utils/localization/build_locale_nsi.py

```diff
@@ -0,0 +1,232 @@
+##############################################################################
+#
+#  PROJECT:     Multi Theft Auto v1.0
+#  LICENSE:     See LICENSE in the top level directory
+#  FILE:        utils/build_locale_nsi.py
```

Comment:
Nit: Use correct filepath or alternatively let's remove it. Let's also fix this for https://github.com/multitheftauto/mtasa-blue/blob/5ecc9c731046c4e5392e483d2c6769ab4939a5c9/utils/localization/build_gettext_catalog_nsi.py#L5 and https://github.com/multitheftauto/mtasa-blue/blob/5ecc9c731046c4e5392e483d2c6769ab4939a5c9/utils/localization/build_gettext_catalog.py#L5 please.
```suggestion
#  FILE:        utils/localization/build_locale_nsi.py
```